src/code/z_play.c

```diff
@@ -1029,7 +1029,7 @@ void Gameplay_Update(GlobalContext* globalCtx) {
     }
 
     func_80070C24(globalCtx, &globalCtx->envCtx, &globalCtx->lightCtx, &globalCtx->pauseCtx, &globalCtx->msgCtx,
-                  &globalCtx->gameOverCtx, globalCtx->state.gfxCtx);
+                  &globalCtx->gameOverCtx.state, globalCtx->state.gfxCtx);
```

Comment:
unintended change?
```suggestion
                  &globalCtx->gameOverCtx, globalCtx->state.gfxCtx);
```

Reply:
yea you should revert this, this is expecting a pointer to the whole context